src/general/GameWorld.cs

```diff
@@ -47,10 +51,21 @@ public class GameWorld : ISaveLoadable
     ///   Creates a new world
     /// </summary>
     /// <param name="settings">Settings to generate the world with</param>
-    public GameWorld(WorldGenerationSettings settings) : this()
+    /// <param name="startingSpecies">Starting species for the player</param>
+    public GameWorld(WorldGenerationSettings settings, Species? startingSpecies = null) : this()
     {
         WorldSettings = settings;
-        PlayerSpecies = CreatePlayerSpecies();
+
+        if (startingSpecies == null)
```

Comment:
Good point. Can apply to Auto-evo Exploring Tool as well. - New Issue?

Reply:
Well isn't the case in the exploring tool that there is *always* the player species present? Which we determined to be actually required to make the "enter game world" button to work sensibly there as there needs to be a player species to control.

Reply:
Well my first design was that you can choose any species and enter the game as it.

Reply:
So it does make sense in general, then to fix the case with no starting species.